packages/frontend/src/hooks/useProjects.ts

```diff
@@ -12,20 +12,27 @@ import {
 import { lightdashApi } from '../api';
 import { useOrganization } from './organization/useOrganization';
 import useToaster from './toaster/useToaster';
+import { useProjectUuid } from './useProjectUuid';
 
-const getProjectsQuery = async () =>
+const getProjectsQuery = async (options?: {
+    headers?: Record<string, string>;
+    projectUuid?: string;
+}) =>
     lightdashApi<OrganizationProject[]>({
-        url: `/org/projects`,
+        url: options?.projectUuid
+            ? `/org/projects?projectUuid=${options.projectUuid}`
+            : `/org/projects`,
         method: 'GET',
-        body: undefined,
+        headers: options?.headers,
```

Comment:
do we need this anymore ?

Reply:
Nice catch. I tried getting all these from earlier.